public/locales/en/translation.json

```diff
@@ -229,538 +229,538 @@
-        "properties.themes": "Thématiques",
-        "properties.keywords": "Mots-clés",
-        "properties.landscape-entities": "Entités paysagères",
-        "related": "Documents associés",
-        "related.document": "Fichier",
+        "properties.historial-data": "Historial data",
+        "properties.cultural-references": "Cultural references",
+        "properties.themes": "Themes",
+        "properties.keywords": "Keywords",
+        "properties.landscape-entities": "Landscape entities",
+        "related": "Linked documents",
+        "related.document": "File",
         "related.key": "Type",
-        "related.label": "Libellé",
+        "related.label": "Label",
         "point.coordinates[0]": "Longitude",
         "point.coordinates[1]": "Latitude"
       },
       "tabs": {
-        "general": "Informations principales",
-        "repeat": "Rephotographie",
-        "landscape": "Paysage",
-        "pictures": "Photographies"
+        "general": "Main informations",
+        "repeat": "Repeat",
+        "landscape": "Landscape",
+        "pictures": "Pictures"
       }
     },
     "picture": {
       "fields": {
-        "label": "Nom du point de vue",
-        "properties.altitude": "Altitude",
-        "properties.meteo": "Conditions météo",
-        "properties.hauteur": "Hauteur de l'œil",
-        "properties.orientation": "Orientation",
-        "properties.focale_35mm": "Focale 35 mm",
-        "properties.focale_objectif": "Focale de l'objectif",
-        "properties.camera_brand": "Marque de l'appareil photo",
-        "properties.camera_model": "Modèle de l'appareil photo",
+        "label": "Viewpoint label",
+        "properties.altitude": "Elevation",
+        "properties.meteo": "Weather",
+        "properties.hauteur": "Eye elevation",
+        "properties.orientation": "Direction",
+        "properties.focale_35mm": "Lens 35 mm",
+        "properties.focale_objectif": "Camera lens",
+        "properties.camera_brand": "Camera brand",
+        "properties.camera_model": "Camera model",
         "properties.observations": "Observations",
         "properties.geometry.coordinates[0]": "Longitude",
         "properties.geometry.coordinates[1]": "Latitude",
-        "properties.index": "Numéro d'ordre",
-        "properties.state": "Statut",
-        "viewpoint": "Point de vue",
-        "owner_id": "ID du photographe",
-        "owner.properties.name": "Photographe",
-        "file": "Photographie",
-        "file.thumbnail": "Photographie",
-        "time": "Heure"
+        "properties.index": "Ordering index",
+        "properties.state": "Status",
+        "viewpoint": "Viewpoint",
+        "owner_id": "Owner ID",
+        "owner.properties.name": "Owner",
+        "file": "Picture",
+        "file.thumbnail": "Picture",
+        "time": "Hour"
       },
       "tabs": {
-        "metadata": "Métadonnées",
-        "repeat": "Rephotographie"
+        "metadata": "Meta data",
+        "repeat": "Repeat"
```

Comment:
The `repeat` key for `Rephotographie/Rephotography` seems a bit odd.
btw, i think the value should be `Rephotography`